plugins/experimental/memcache/tsmemcache.cc

```diff
@@ -249,7 +249,7 @@ MC::add_binary_header(uint16_t err, uint8_t hdr_len, uint16_t key_len, uint32_t
 int
 MC::write_binary_error(protocol_binary_response_status err, int swallow)
 {
-  const char *errstr = "Unknown error";
+  const char *errstr;
```

Comment:
Let's initialize to `nullptr` while we're at it.

Reply:
Thanks! Fixed at [afe6f76](https://github.com/apache/trafficserver/pull/12226/commits/afe6f767418cd7828d3aadcaef06fa63073ee1f9).